apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/[keyId]/settings/update-key-remaining.tsx

```diff
@@ -219,34 +167,40 @@ export const UpdateKeyRemaining: React.FC<Props> = ({ apiKey }) => {
                         className="w-full"
                         type="number"
                         {...field}
-                        value={form.watch("refill.interval") === "none" ? undefined : field.value}
+                        value={form.getValues("limitEnabled") ? field.value : undefined}
```

Comment:
`getValues` won't get updated when `limitEnabled` changes, so that might cause some issues. Just giving heads up.